spring-cloud-dataflow-module-deployers/spring-cloud-dataflow-module-deployer-kubernetes/README.md

```diff
@@ -0,0 +1,176 @@
+SPI implementation for deploying Spring Cloud Stream modules to [Kubernetes](http://http://kubernetes.io/).
+
+### Setup a Kubernetes Cluster 
+
+If you don't have access to a Kubernetes Cluster, you can deploy one locally.
+This requires a local functional Docker installation and the usage of some 
+nice scripts to setup [Kubernetes](https://github.com/olmoser/docker-compose-kubernetes).
+
+In a nutshell, do the following steps:
+```
+$ git clone https://github.com/olmoser/docker-compose-kubernetes
+$ cd docker-compose-kubernetes
+$ docker-machine create --driver virtualbox sc-dataflow-dev 
+$ docker-machine start sc-dataflow-dev 
+$ eval "$(docker-machine env sc-dataflow-dev)"
+$ ./kube-up.sh
+```
+
+Once the cluster is up, configure you local system to your local DNS server `10.0.0.10` 
```

Comment:
Ok, that was bad phrasing. A kubernetes cluster comes with a DNS service. If you add the Kubernetes DNS `10.0.0.10` to your local DNS config (e.g. /etc/resolv.conf on Linux), you can resolve all Kubernetes provisioned services from your development machine (needed later in the Readme).